fixtures/load-repo.js

```diff
@@ -0,0 +1,205 @@
+        // Files with more node types tend to be more interesting and better candidates for lint testing
+        parsed = await babel.parseAsync(contents, {
+          parserOpts: {
+            plugins: [
+              'jsx',
+              'classProperties',
+              'dynamicImport',
+              'nullishCoalescingOperator',
+              'optionalChaining',
+            ],
+          },
+          filenameRelative,
+        });
+      } catch (error) {
+        return invalidResult;
+      }
+
+      babel.traverse(parsed, {
+        enter(path) {
+          numNodes++;
+          seenNodeTypes.add(path.type);
+        },
+      });
+
+      const numNodeTypes = seenNodeTypes.size;
+
+      let eslintReport;
+      try {
+        eslintReport = eslint.executeOnText(contents).results[0];
+      } catch (error) {
+        console.log('eslint error', file, error);
+        return invalidResult;
+      }
+
+      const numLintErrors = eslintReport.errorCount + eslintReport.warningCount;
+      const eslintFixedContents = eslintReport.output || contents;
+
+      let prettifiedContents;
+
+      try {
+        prettifiedContents = prettier.format(eslintFixedContents, prettierOpts);
+      } catch (error) {
+        console.log('prettier error', file, error);
+        return invalidResult;
+      }
+
+      // Generates a "score" to a file based on the # of nodes, # of node types
+      // Used for sorting
+      // NumNodeTypes is weighted much more strongly
+      // because files tend to have a lot of nodes compared to the number of node types
+      const score = numNodeTypes * 10 + numNodes / 1000 - numLintErrors * 30;
+      return {
+        file,
+        contents: prettifiedContents,
+        score,
+        numLintErrors,
+        numNodeTypes,
+        loc: contents.split('\n').length,
+      };
+    })
+  );
+
+  const sortedFiles = filesWithNumNodeTypes
+    // Make sure it parsed correctly, has a reasonable number of errors, and is not too short
+    .filter(
+      (file) => file.score > 0 && file.numLintErrors < 100 && file.loc > 50
+    )
+    .sort((a, b) => b.score - a.score)
+    .slice(0, 50);
+
+  const { chosenFiles } = await prompts({
+    name: 'chosenFiles',
+    type: 'multiselect',
+    choices: sortedFiles.map((file) => ({
+      title: file.file,
+      value: file,
+      description: `${file.numLintErrors} lint errors, ${file.loc} loc, ${file.numNodeTypes} node types`,
+    })),
+    message: 'Choose files to import',
+  });
+
+  await Promise.all(
+    chosenFiles.map(async ({ file, contents }) => {
+      const outPath = path.join(fixturesDir, file);
+      await mkdir(path.dirname(outPath));
+      await writeFileAsync(outPath, contents);
+    })
+  );
+
+  console.log(kleur.bold().blue('Imported files'));
+
+  await runCommand('npm', ['run', 'lint']).catch((error) =>
+    // eslint-disable-next-line no-process-exit
+    process.exit(error)
```

Comment:
I wonder if it might be good to add a message here about how to deal with errors (i.e., some of the stuff you put in the README - fixing, commenting, or removing files, or adding suggestions for updates to the config).

Reply:
Good idea 👍

Reply:
This comment is really the same as https://github.com/cloudfour/eslint-config/pull/91#discussion_r401287200, so you can just resolve it.